packages/wallets/src/hooks/usePasswordValidation.tsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from 'react';
+import zxcvbn from 'zxcvbn';
+
+const usePasswordValidation = (password: string) => {
+    const [errorMessage, setErrorMessage] = useState('');
+    const [score, setScore] = useState(0);
+
+    useEffect(() => {
+        const { feedback } = zxcvbn(password);
+
+        if (!/^.{8,25}$/.test(password)) {
+            setErrorMessage('You should enter 8-25 characters.');
+        } else if (!/^(?=.*[a-z])(?=.*[A-Z])(?=.*\d).+$/.test(password)) {
+            setErrorMessage('Password should have lower and uppercase English letters with numbers.');
+        } else {
+            setErrorMessage(feedback.warning);
+        }
+    }, [password]);
+
+    useEffect(() => {
+        const hasNumber = /\d/.test(password);
+        const hasUpper = /[A-Z]/.test(password);
+        const hasLower = /[a-z]/.test(password);
+        const isLengthValid = /^.{8,25}$/.test(password);
+        const hasSymbol = /\W/.test(password);
+        const hasMoreThanOneSymbol = (password.match(/\W/g) || []).length > 1;
```

Comment:
@aizad-deriv small comment, but sonarCloud wants ?? instead of ||

Reply:
yes please fix 👍🏼